docker/Dockerfile

```
@@ -85,44 +84,22 @@ RUN python setup.py bdist_wheel
 FROM base-runtime AS wheel-lightning-qubit
 COPY --from=build-wheel-lightning-qubit /opt/pennylane-lightning/dist/ /
 RUN pip install --force-reinstall --no-cache-dir PennyLane_Lightning*.whl && rm PennyLane_Lightning*.whl
-RUN pip install --no-cache-dir -U pennylane
-
-# Download Kokkos release and build OpenMP backend
-FROM base-build as build-kokkos-openmp
-WORKDIR /opt/kokkos-${KOKKOS_VERSION}
-RUN wget --progress=dot:giga https://github.com/kokkos/kokkos/archive/refs/tags/${KOKKOS_VERSION}.tar.gz
-RUN tar -zxf ${KOKKOS_VERSION}.tar.gz && mv /opt/kokkos-${KOKKOS_VERSION}/kokkos-${KOKKOS_VERSION}/* /opt/kokkos-${KOKKOS_VERSION}
-RUN cmake -S /opt/kokkos-${KOKKOS_VERSION} -B /opt/kokkos-${KOKKOS_VERSION}/build \
-    -G Ninja \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_INSTALL_PREFIX=/opt/kokkos \
-    -DBUILD_SHARED_LIBS:BOOL=ON \
-    -DBUILD_TESTING:BOOL=OFF \
-    -DKokkos_ENABLE_SERIAL:BOOL=ON \
-    -DKokkos_ENABLE_OPENMP:BOOL=ON \
-    -DCMAKE_CXX_STANDARD=20 \
-    -DKokkos_ENABLE_EXAMPLES:BOOL=OFF \
-    -DKokkos_ENABLE_TESTS:BOOL=OFF \
-    -DKokkos_ENABLE_LIBDL:BOOL=OFF
-RUN cmake --build /opt/kokkos-${KOKKOS_VERSION}/build
-RUN cmake --install /opt/kokkos-${KOKKOS_VERSION}/build
+RUN pip install --no-cache-dir git+https://github.com/PennyLaneAI/pennylane.git@${PENNYLANE_VERSION}
 
 # Download Lightning release and build lightning-kokkos backend with Kokkos-OpenMP
 FROM base-build-python AS build-wheel-lightning-kokkos-openmp
 WORKDIR /opt/pennylane-lightning
-COPY --from=build-kokkos-openmp /opt/kokkos /opt/kokkos
-ENV CMAKE_PREFIX_PATH=/opt/kokkos:$CMAKE_PREFIX_PATH
+ENV PL_BACKEND=lightning_kokkos
 RUN pip uninstall -y pennylane-lightning
-RUN PL_BACKEND=lightning_kokkos python setup.py build_ext
-RUN PL_BACKEND=lightning_kokkos python setup.py bdist_wheel
+RUN python setup.py build_ext -i --define="PL_BACKEND=lightning_kokkos;Kokkos_ENABLE_SERIAL:BOOL=ON;Kokkos_ENABLE_OPENMP:BOOL=ON"
```

This seems not be required as `PL_BACKEND` is already set to LK (line 92) 
```suggestion
RUN python setup.py build_ext -i --define="Kokkos_ENABLE_SERIAL:BOOL=ON;Kokkos_ENABLE_OPENMP:BOOL=ON"
```